<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEncodeWidget</class>
 <widget class="QWidget" name="TextEncodeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="inputLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Enter text:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="inputText"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="outputLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Select encoding:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="encodingOptionsComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Output:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="outputText"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
